modules/update-upgrading-web.adoc

```diff
@@ -15,6 +15,29 @@ ifeval::["{context}" == "updating-cluster-rhel-compute"]
 endif::[]
 
 [id="update-upgrading-web_{context}"]
+= Optional: Changing the update server by using the web console
```

Comment:
Same comment about creating a new file so that a module does not have two procedures.  Also same comment about hopefully finding a way to avoid "you can use it."